examples/news_alerts/kafka_utils.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/log;
+import ballerina/uuid;
+import ballerinax/kafka;
+
+configurable decimal POLL_INTERVAL = 100.0;
+
+type NewsProducerRecord record {|
+    *kafka:AnydataProducerRecord;
+    string key;
+    NewsRecord value;
+|};
+
+type NewsConsumerRecord record {|
+    *kafka:AnydataConsumerRecord;
+    NewsRecord value;
+|};
+
+final kafka:Producer producer = check new (kafka:DEFAULT_URL);
+
+isolated function produceNews(NewsUpdate newsUpdate) returns NewsRecord|error {
+    string id = uuid:createType1AsString();
+    Agency agency = check getAgency(newsUpdate.publisherId);
+    NewsRecord news = {id: id, ...newsUpdate};
+    NewsProducerRecord producerRecord = {
+        topic: agency,
+        key: id,
+        value: news
+    };
+    check producer->send(producerRecord);
+    return news;
+}
+
+isolated class NewsStream {
+    private final string consumerGroup;
+    private final Agency agency;
+    private final kafka:Consumer consumer;
+
+    isolated function init(string consumerGroup, Agency agency) returns error? {
+        self.consumerGroup = consumerGroup;
+        self.agency = agency;
+        kafka:ConsumerConfiguration consumerConfiguration = {
+            groupId: consumerGroup,
+            offsetReset: "earliest",
+            topics: [agency],
```

Comment:
Can't we do `topics: agency`? @dilanSachi

Reply:
Ah yes. Both `string` and `string[]` is allowed

Reply:
Yes, we can. Updated.